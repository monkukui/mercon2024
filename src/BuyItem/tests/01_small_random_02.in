4
165704716 613085206 3
696197149 567409403 61
713430567 480971831 77
989592817 35704758 69
