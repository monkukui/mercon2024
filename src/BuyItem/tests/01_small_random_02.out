4
35704758 989592817
480971831 713430567
567409403 696197149
613085206 165704716
